### DISCUSSION TITLE
Output 2048x2048

### DISCUSSION THREAD

**Discussion by LouisDeconinck** [lllyasviel/Fooocus#425] [General]
How can I have fooocus output 2048x2048 or have it upscale all images automatically to that resolution?

**Comment by barepixels** [comment]
automatically upscale? not possible

UPDATE:
I was curious if my 3070 8GB card could do 2048 so with Fooocus-MRE I set a custom resolution of 2048x2048.
1) The good news is my 8GB video card can do it without the need for tiling or running out of memory
2) The bad news is SDLX is based on 1024 so it adds additional subjects and distortion

![2048](https://github.com/lllyasviel/Fooocus/assets/92839033/5193ae78-b6a9-4496-a1d8-df7a7728c465)
